packages/graphql/tests/integration/utils/tests-helper.ts

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Cypher from "@neo4j/cypher-builder";
+import type { ExecutionResult, GraphQLArgs } from "graphql";
+import { graphql as graphqlRuntime } from "graphql";
+import * as neo4j from "neo4j-driver";
+import { Memoize } from "typescript-memoize";
+import type { Neo4jGraphQLConstructor, Neo4jGraphQLContext } from "../../../src";
+import { Neo4jGraphQL } from "../../../src";
+import { createBearerToken } from "../../utils/create-bearer-token";
+import { UniqueType } from "../../utils/graphql-types";
+
+const INT_TEST_DB_NAME = "neo4jgraphqlinttestdatabase";
+const DEFAULT_DB = "neo4j";
+
+let helperLock = false;
+
+export class TestHelper {
+    private database: string = DEFAULT_DB;
+    private neo4jGraphQL: Neo4jGraphQL | undefined;
+    private uniqueTypes: UniqueType[] = [];
+
+    public createBearerToken(secret: string, extraData?: Record<string, any>) {
+        return createBearerToken(secret, extraData);
+    }
+
+    public async initNeo4jGraphQL(options: Omit<Neo4jGraphQLConstructor, "driver">): Promise<Neo4jGraphQL> {
+        if (helperLock) {
+            throw new Error("TestHelper still open. Did you forget calling .close()?");
+        }
+        helperLock = true;
+        if (this.neo4jGraphQL) {
+            throw new Error("Neo4jGraphQL already initialized");
+        }
+        const driver = await this.getDriver();
+        this.neo4jGraphQL = new Neo4jGraphQL({
```

Comment:
Nitpick: Maybe we set the validate as false by default to reduce the load and improve the tests speed

Reply:
mmm, we could discuss that, however a quick test show that the performance improvements is less than 1% which is well under the margin of error so I think the impact is negligible at the moment

Reply:
Not important anyway, as could be added with ease afterward, but I remember that when you have some type-dependant directives you will have a larger performance impact, although this could have changed between releases!